### PR TITLE
chore: improvement session and event mv performance

### PR DESCRIPTION
**Pull request by llmin**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

**Review by amliuyong — APPROVED** [review]
LGTM!

**Review by zxkane — CHANGES_REQUESTED** [review]

File: src/analytics/lambdas/load-data-workflow/refresh-views.ts [hunk 1]
```diff
@@ -39,32 +42,39 @@ export const handler = async (_e: any, _c: Context) => {
...
+
+        const sqlStatements = sqlStatementForApp.split(';').map(s => s.trim()).filter(s => s.length > 0);
+        logger.info('sqlStatements', { sqlStatements });
+        const queryId = await executeStatements(
+          redshiftDataApiClient, sqlStatements, redshiftProps.serverlessRedshiftProps, redshiftProps.provisionedRedshiftProps);
+        if (queryId) {
+          queryIds.push(queryId);
+        }
+
+        logger.warn(`Refresh mv for app: ${schema} finished`);
```
[the remark this answers is not in the document]
Comment (llmin) [reply src/analytics/lambdas/load-data-workflow/refresh-views.ts:72]: Want tho show mv refresh message on default log level 'WARN'

File: docs/zh/analytics/dashboard/retention.md [hunk 2]
```diff
@@ -11,32 +11,32 @@
...
-        --8<-- "src/analytics/private/sqls/athena/clickstream-lifecycle-weekly-query.sql"
-        ```
-        ```sql title="clickstream-lifecycle-daily-query.sql"
-        --8<-- "src/analytics/private/sqls/athena/clickstream-lifecycle-daily-query.sql"
-        ```
-        ```sql title="clickstream-lifecycle-daily-query.sql"
-        --8<-- "src/analytics/private/sqls/athena/clickstream-retention-query.sql"
-        ```
+??? 示例"SQL 命令"
+     === "Redshift"
```
Comment (zxkane) [inline docs/zh/analytics/dashboard/retention.md:20]: Did you have a local running after making this change?

File: src/analytics/lambdas/load-data-workflow/refresh-views.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (zxkane) [reply src/analytics/lambdas/load-data-workflow/refresh-views.ts:72]: I'm against it. It's by design behavior, not using `warn` for logging.

[the remark this answers is not in the document]
Comment (llmin) [reply src/analytics/lambdas/load-data-workflow/refresh-views.ts]: change to info

File: docs/zh/analytics/dashboard/retention.md (hunk 2 above)
Comment (zxkane) [reply docs/zh/analytics/dashboard/retention.md:20]: You have to revert those format changes. It breaks the code rendering.

image

Comment (llmin) [reply docs/zh/analytics/dashboard/retention.md]: verified

**Review by zxkane — APPROVED** [review]

**Pull request merged**